text/0000-any-stack-buildpacks.md

```diff
@@ -11,7 +11,7 @@
 # Summary
 [summary]: #summary
 
-This change removes the restriction that buildpacks must specify at least one stack in `buildpack.toml`. When buildpacks specify no stacks, the lifecycle will assume that the buildpack is valid on all stacks.
+This change alls buildpack authors to explicitly remove the restriction that buildpacks must specify at least one stack in `buildpack.toml`. When buildpacks specify a wildcard stack value, the lifecycle will assume that the buildpack is valid on all stacks.
```

Comment:
```suggestion
This change allows buildpack authors to explicitly remove the restriction that buildpacks must specify at least one stack in `buildpack.toml`. When buildpacks specify a wildcard stack value, the lifecycle will assume that the buildpack is valid on all stacks.
```

Reply:
Not a blocker, but isn't specify "all stacks" specifying at least one stack? :)